system-addon/lib/ActivityStream.jsm

```diff
@@ -4,25 +4,25 @@
 "use strict";
 
 const {utils: Cu} = Components;
```

Comment:
It seems like there are a bunch of cases where this line can be removed. Did you want to do that in this patch or file as a follow-up?

Reply:
k88hudson: sounds like there's going to be a mozilla-central-wide cleanup of now-unnecessary Components aliasing, so let's just wait for that clean up.

https://groups.google.com/forum/#!topic/firefox-dev/CeRsSWNDMjc
https://bugzilla.mozilla.org/show_bug.cgi?id=1432992